shell/platform/android/android_context_gl_impeller.cc

```diff
@@ -92,7 +92,7 @@ AndroidContextGLImpeller::AndroidContextGLImpeller(
   }
 
   impeller::egl::ConfigDescriptor desc;
-  desc.api = impeller::egl::API::kOpenGLES2;
+  desc.api = impeller::egl::API::kOpenGLES3;
```

Comment:
Is this valid to do? Defiintely outside of my wheelhouse. - we do a lot of GLES 2 stuff, do we need to worry about GLES 3 having different semantics?

@chinmaygarde

Reply:
Nope. That's how you'd do it. ES3 is backwards compatible with the same sematics.

Reply:
Coolio